website/content/docs/sync/gcpsm.mdx

```diff
@@ -0,0 +1,249 @@
+  to provision this Service Account via Terraform can be found below.
+
+1. Configure a sync destination with the Service Account JSON credentials created in the previous step. See docs for
+  [alternative ways](/vault/docs/secrets/gcp#authentication) to pass in the `credentials` parameter.
+
+  ```shell-session
+  $ vault write sys/sync/destinations/gcp-sm/my-dest \
+      credentials='@path/to/credentials.json'
+  ```
+
+  **Output:**
+
+  <CodeBlockConfig hideClipboard>
+
+  ```plaintext
+  Key                   Value
+  ---                   -----
+  connection_details    map[credentials:*****]
+  name                  my-dest
+  type                  gcp-sm
+  ```
+
+  </CodeBlockConfig>
+
+## Usage
+
+1. If you do not already have a KVv2 secret to sync, mount a new KVv2 secrets engine.
+
+  ```shell-session
+  $ vault secrets enable -path='my-kv' kv-v2
+  ```
+
+  **Output:**
+
+  <CodeBlockConfig hideClipboard>
+
+  ```plaintext
+  Success! Enabled the kv-v2 secrets engine at: my-kv/
+  ```
+
+  </CodeBlockConfig>
+
+1. Create secrets you wish to sync with a target GCP Secret Manager.
+
+  ```shell-session
+  $ vault kv put -mount='my-kv' my-secret foo=bar
+  ```
+
+  **Output:**
+
+  <CodeBlockConfig hideClipboard>
+
+  ```plaintext
+  ==== Secret Path ====
+  my-kv/data/my-secret
+
+  ======= Metadata =======
+  Key                Value
+  ---                -----
+  created_time       <timestamp>
+  custom_metadata    <nil>
+  deletion_time      n/a
+  destroyed          false
+  version            1
+  ```
+
+  </CodeBlockConfig>
+
+1. Create an association between the destination and a secret to synchronize.
+
+  ```shell-session
+  $ vault write sys/sync/destinations/aws-sm/my-dest/associations/set \
+      mount='my-kv' \
+      secret_name='my-secret'
+  ```
+
+  **Output:**
+
+  <CodeBlockConfig hideClipboard>
+
+  ```plaintext
+  Key                   Value
+  ---                   -----
+  associated_secrets    map[kv_1234/my-secret:map[accessor:kv_1234 secret_name:my-secret sync_status:SYNCED updated_at:<timestamp>]]
+  store_name            my-dest
+  store_type            gcp-sm
+  ```
+
+  </CodeBlockConfig>
+
+1. Navigate to the [Secret Manager](https://console.cloud.google.com/security/secret-manager) in the Google Cloud console
+  to confirm your secret was successfully created in your GCP project.
+
+Moving forward, any modification on the Vault secret will be propagated in near real time to its GCP Secret Manager
+counterpart. Creating a new secret version in Vault will create a new version in GCP Secret Manager. Deleting the secret
+or the association in Vault will delete the secret in your GCP project as well.
+
+## Permissions
+
+The credentials given to Vault must have the following permissions to synchronize secrets:
+
+```shell-session
+secretmanager.secrets.create
+secretmanager.secrets.delete
+secretmanager.secrets.get
+secretmanager.secrets.list
```

Comment:
I don't _think_ the read or list accesses are needed? I put the minimal amount of permissions in the AWS page,

Reply:
I think you are correct but I'd leave them here since we are very close to the deadline, unless you are able to confirm 100% it works without.